web/profiles/webspark/webspark/webspark.install

```diff
@@ -735,17 +735,27 @@ function webspark_update_10012(&$sandbox): void{
   \Drupal::state()->set('configuration_locked', TRUE);
 }
 
+/**
+ * WS2-1623 - Enable the asu_config_utility module.
+ */
+function webspark_update_10013(&$sandbox): void {
+  \Drupal::service('module_installer')->install(['asu_config_utility']);
+  \Drupal::service('module_installer')->install(['asu_components_appearance']);
+}
+
```

Comment:
Update hook here to enable the new module is great.

The old historical updates are going to be problematic for older sites updating that haven't hit this yet. Two possible solves: 
1. add a helper function to the profile that can be used to check if the new module is enabled and does so if it isn't and then add that to the start of all the historical updates. OR
2. keep the historical config update service in place and don't update the past update hooks, and we'll have some duplicated and deprecated code for a bit, but we can remove it eventually. All new code can use the new asu config utility moving forward.

Reply:
@mlsamuelson I have decided to go with option #2.